src/libraries/Directory.Build.targets

```diff
@@ -22,6 +22,7 @@
     <StrongNameKeyId Condition="'$(IsTestProject)' == 'true'">$(TestStrongNameKeyId)</StrongNameKeyId>
   </PropertyGroup>
 
+  <Import Project="$(RepositoryEngineeringDir)intellisense.targets" Condition="'$(IsTestProject)' != 'true'" />
```

Comment:
Why do we don't need this for TestProjects? Test projects can also cross target, so I guess intellisense is broken there as well.

Reply:
Yes it’s broken there as well, I am working on fixing it. But currently I have a road block there in resolving package references. I will be fixing that as well